src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/SlingInitialContentBundleEntryMetaDataCollector.java

```diff
@@ -106,10 +132,54 @@ Set<SlingInitialContentBundleEntryMetaData> collectFromContextAndWriteTmpFiles()
                 }
             }
         }
-
+        
         return collectedSlingInitialContentBundleEntries;
     }
+    
+    Set<SlingInitialContentBundleEntryMetaData> getDefaultContentXmlEntries() throws UnsupportedEncodingException {
+        
+        createDefaultContentXMLFiles();
+        return defaultContentXmlEntries;
+        
+    }
+
+    private void createDefaultContentXMLFiles() throws UnsupportedEncodingException {
```

Comment:
`UnsupportedEncodingException` is never thrown if you removed the unnecessary throws clause from `addDefaultSlingContentXMLAndParents`